cmd/file_render.go

```diff
@@ -27,8 +27,19 @@ func executeFileRenderCmd(_ *cobra.Command, _ []string) error {
 func newFileRenderCmd() *cobra.Command {
 	renderCmd := &cobra.Command{
 		Use:   "render",
-		Short: "Render the configuration as Kong declarative config",
-		Long:  ``,
+		Short: "Combines multiple complete configuration files into one Kong declarative config file.",
+		Long:  `Combines multiple complete configuration files into one Kong
+	declarative config file.
+
+ This command can render the output in JSON or YAML format. Unlike 
+ "deck file merge", the render command accepts complete configuration files, 
+ while "deck file merge" only combines partial file snippets.
+
+ For example, the following command takes two input files and renders them as one 
+ combined JSON file:
+
+   deck file render kong1.yml kong2.yml -o kong3 --format json
```

Comment:
```suggestion
   deck file render kong1.yml kong2.yml -o kong3.json --format json
```
Should be more accurate

Reply:
I tried both ways, and they both worked. I _also_ accidentally tried `kong3.yml --format json` and ended up with a file named kong3.yaml but the contents were json. Should there be any error or something around that 🤔

Reply:
@Tieske is it an expected behavior?

Reply:
@lena-larionova disregard